docs/source/performance.mdx

```diff
@@ -388,10 +430,26 @@ Note: tf32 mode is internal to CUDA and can't be accessed directly via `tensor.t
 
 Note: you need `torch>=1.7` to enjoy this feature.
 
+You can also see a variety of benchmarks on tf32 vs other precisions:
+[RTX-3090](https://github.com/huggingface/transformers/issues/14608#issuecomment-1004390803) and
+[A100](https://github.com/huggingface/transformers/issues/15026#issuecomment-1004543189).
+
+
+
+### Gradient Accumulation
+
+Besides using gradient accumulation when a larger batch size is needed and can't normally be fit into the gpu's memory, enabling can also provide a 20-30% speed up as it runs the optimizer less often, e.g. see benchmarks for [RTX-3090](https://github.com/huggingface/transformers/issues/14608#issuecomment-1004392537)
+and [A100](https://github.com/huggingface/transformers/issues/15026#issuecomment-1004592231).
+
+To activate this feature in 🤗 Trainer add `--gradient_accumulation_steps 4` to its arguments (experiment with the value to get the best performance.
+
+It's important to remember that using gradient accumulation you may end up with a much larger effective batch size, so you may need to adjust the learning rate, its warm up and for very short datasets it'll impact the loss as the training will end up doing less steps than normal.
```

Comment:
I don't think this comment is accurate. Using gradient accumulation is the same as using a larger batch size: so you're not getting a speedup because of it (compared to the same effective batch size), it just allows you to run the training at the batch size you want.

Reply:
could you please be more specific which part(s) you think are inaccurate? The effective batch size?

Reply:
> enabling can also provide a 20-30% speed up as it runs the optimizer less often

That is not accurate. Using a larger batch size gives the speedup, not gradient accumulation per se.

Reply:
Oh, I see what I mean now. How about changing the first para to:

> Since gradient accumulation essentially is identical to having a larger batch size, just as with the larger batch size here you you are likely to see a 20-30% speed up due to the optimizer running less often.